src/ring_theory/polynomial/cyclotomic.lean

```diff
@@ -451,6 +451,72 @@ begin
   cyclotomic'_eq_X_pow_sub_one_div hpos hz, finset.prod_congr (refl k.proper_divisors) h]
 end
 
+lemma eq_cyclotomic_iff {R : Type*} [comm_ring R] [nontrivial R] {n : ℕ} (hpos: 0 < n)
+  (P : polynomial R) :
+  P = cyclotomic n R ↔ P * (∏ i in nat.proper_divisors n, polynomial.cyclotomic i R) = X ^ n - 1 :=
+begin
+  split,
+  { intro hcycl,
+    rw [hcycl, ← finset.prod_insert (@nat.proper_divisors.not_self_mem n),
+      ← nat.divisors_eq_proper_divisors_insert_self_of_pos hpos],
+    exact prod_cyclotomic_eq_X_pow_sub_one hpos R },
+  { intro hP,
+    have prod_monic : (∏ i in nat.proper_divisors n, cyclotomic i R).monic,
+    { apply monic_prod_of_monic,
+      intros i hi,
+      exact cyclotomic.monic i R },
+    rw [@cyclotomic_eq_X_pow_sub_one_div R _ _ _ hpos,
+      (div_mod_by_monic_unique P 0 prod_monic _).1],
+    split,
+    { rwa [zero_add, mul_comm] },
+    rw [degree_zero, bot_lt_iff_ne_bot],
+    intro h,
+    exact monic.ne_zero prod_monic (degree_eq_bot.1 h) },
+end
+
+/-- If `p` is prime, then `cyclotomic p R = geom_series X p`. -/
+lemma cyclotomic_of_prime {R : Type*} [comm_ring R] [nontrivial R] {p : ℕ} (hp : nat.prime p) :
+  geom_series X p = cyclotomic p R :=
+begin
+  apply (@eq_cyclotomic_iff R _ _ p (nat.prime.pos hp) _).2,
+  simp only [nat.prime.proper_divisors hp, geom_sum_mul, finset.prod_singleton, cyclotomic_one],
+end
+
+/-- The constant term of `cyclotomic n R` is `1` if `2 ≤ n`. -/
+lemma cyclotomic_coeff_zero {R : Type*} [comm_ring R] (n : ℕ) :
+  2 ≤ n → (cyclotomic n R).coeff 0 = 1 :=
+begin
+  apply nat.strong_induction_on n,
+  intros k hi hk,
```

Comment:
The `induction` tactic can do both these lines at the same time:
```suggestion
lemma cyclotomic_coeff_zero {R : Type*} [comm_ring R] (n : ℕ) (hk : 2 ≤ n) :
  (cyclotomic n R).coeff 0 = 1 :=
begin
  induction n using nat.strong_induction_on with k hi,
```

Reply:
Yes, but I really use strong induction. AFAIK there is no `strong_induction` tactic...

Reply:
That's what `using` is for!

Reply:
(I tested the above and it worked)

Reply:
Very nice! I still have to `intro hk`, but that's inevitable I think.

Reply:
No you don't, the above diff does that for you

Reply:
Even better!